system-config/select-apt-mirror.sh

```diff
@@ -20,7 +20,7 @@
 # This script changes the apt mirror to use.
 
 if [[ "$1" -ne "" ]]; then
-    releaseName=$1
+    releaseName="$1"
```

Comment:
Should the documentation be updated as well? Are we also changing which release we are using in this script?